mosviz/plugins/cutout_tool/cutout_tool.py

```diff
@@ -755,9 +758,11 @@ def preview(self):
 
         #Make cutouts using info in catalog.
         self.statusBar().showMessage("Making cutouts")
-        hdu = self.make_cutouts(
-            t, self.img_path, programName, clobber=True,
-            apply_rotation=True, ispreview=True)
+        hdu = go_make_cutouts(
```

Comment:
Here the `hdu` should be the first and only cutout in `fits_cutouts` from the `go_make_cutouts` function. I think it may be best to do the indexing (`fits_cutouts[0]`) inside this function instead of `go_make_cutouts`

Reply:
Done.